libs/libjson/_internal/Source/internalJSONNode.h

```diff
@@ -156,8 +156,8 @@ class internalJSONNode {
 	   DECL_CAST_OP(short)
 	   DECL_CAST_OP(int)
 	   DECL_CAST_OP(long)
+	   DECL_CAST_OP(long long)
 	   #ifndef JSON_ISO_STRICT
-		  DECL_CAST_OP(long long)
```

Comment:
We are currently compiling with ISO_STRICT. I did not manage to compile without it. If I turn it off in the json options, it turns it on again because the compiler is instructed to use ANSI. Not sure how to fix this (maybe requires changing the cmake configuration).